plugins.json

```diff
@@ -1,4 +1,91 @@
 [
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+    "icon_data": "",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-calls-v0.4.1.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.4.1",
+    "hosting": "",
+    "author_type": "mattermost",
+    "release_stage": "beta",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmIqBkoACgkQ0bVLR6XO/sTRkBAAlHLVMmd1kFChdtbEPt/wDKwznYAu6NCGXrDMknphNSAs1klK3z3yjN8FMNNNnLogxBb9LfnromvCxqvg59eMwAXrXlZIW/1TE3kfOD/Chp0FBNLwpBoVO3SD4f/3bCk0ItBU+hNij2E3/Q6LImgugN6epVo1vcaEztSzObpc82/UtG+y0dttxDiO+/b45LRcBhF2gcg8ePPsIsaW3v3v6ZbHOoYf+cylO4ZFNb6O/2qhyw/zcfKoMMBPs2ZTRYFAFwv+YgLzY6wrZZthPKwaSKznVWHYd0tjIYnnB8dL+9sM0ckyhT131jBAMn9XhSZcStdB8paEW0ORweCAf3UBr6Gk0ZYdpQNC/5W4lJvaaVRIzQ0p3CrfSV2v54Y7DFMOB1cp/GlPK/jkX16n1TR7gLAI0hVh1fpX6qbLvV0w6f40vYfrbuVd20dZADHNhCcQH7DX3JdVe1pPi2eRVKBZ00qZO5V1KnthzQst3zLkrMC/3iZll0OaM9/RaGRWDaZ1Dd4w2O8f1Yz9foeRCcYyWhIALGsfxmBTs+nx35GvtVIKVeFv4WmM96wgbJZ0nzfO6uvfgyszCkQz8XPKlr/R96bOjCPImsiFq1QC16ZjHxtg69X8EESanMgilESonHKitzZeCiaci3XxSFSf/7Os/KGyIoFQoBEFC527CYFYj3k=",
+    "repo_name": "mattermost-plugin-calls",
+    "manifest": {
+      "id": "com.mattermost.calls",
+      "name": "Calls",
+      "description": "Integrates real-time voice communication in Mattermost",
+      "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+      "support_url": "https://github.com/mattermost/mattermost-plugin-calls/issues",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.4.1",
+      "version": "0.4.1",
+      "min_server_version": "6.3.0",
+      "server": {
+        "executables": {
+          "linux-amd64": "server/dist/plugin-linux-amd64",
+          "darwin-amd64": "server/dist/plugin-darwin-amd64"
```

Comment:
Is this expected? Asking because I thought we were packaging linux only.

Reply:
Yah, it's expected -- it just means there will be an error if you try to install it on mac. I still have to test what this will look like on the user's end (took forever to get to this point :)

Reply:
Sounds good, thanks for confirming :+1: